miso-web/src/main/webapp/WEB-INF/pages/editInstrument.jsp

```diff
@@ -83,7 +83,7 @@
         <div id="records_arrowclick" class="toggleLeft"></div>
       </div>
       <div id="recordsdiv" class="expandable_section" style="display:none;">
-        <miso:list-section id="list_servicerecords" name="Service Records" target="servicerecord" alwaysShow="true" items="${serviceRecords}" config="{instrumentId: ${instrument.id}, retiredInstrument: ${instrument.dateDecommissioned != null && instrument.upgradedInstrument == null}, hasPositions: ${not empty instrument.instrumentModel.positions}, userIsAdmin: ${miso:isAdmin()}}"/>
+        <miso:list-section id="list_servicerecords" name="Service Records" target="servicerecord" alwaysShow="true" items="${serviceRecords}" config="{instrumentId: ${instrument.id}, retired: ${instrument.dateDecommissioned != null && instrument.upgradedInstrument == null}, hasPositions: ${not empty instrument.instrumentModel.positions}, userIsAdmin: ${miso:isAdmin()}}"/>
```

Comment:
What prompted this change?

Reply:
There is a config called `retired` for freezer also. So on `list_servicerecord.js` I can use `config.retired` without the if case for checking whether it is a freezer or instrument

Reply:
Thanks, I see that now!